gradle/eclipse.gradle

```diff
@@ -15,7 +15,8 @@ allprojects {
                     classpath.entries.removeAll { it.path.contains("/subprojects") && it.kind == 'lib' }
                     // Add needed resources for running gradle as a non daemon java application
                     classpath.entries.add(new org.gradle.plugins.ide.eclipse.model.SourceFolder("build/generated-resources/main", null))
-                    classpath.entries.add(new org.gradle.plugins.ide.eclipse.model.SourceFolder("build/generated-resources/test", null))
+                    if (file("build/generated-resources/test").exists())
```

Comment:
In which case it wouldn't exist?

Reply:
Upon initial import in Eclipse, at least with Gradle 4.4-rc5